src/matlab2tikz.m

```diff
@@ -435,11 +435,11 @@ function matlab2tikz(varargin)
 
     % Alternative Positioning of axes.
     % Select relevant Axes and draw them.
-    [relevantAxesHandles, axesBoundingBox] = getRelevantAxes(m2t, axesHandles);
+    [m2t, axesBoundingBox] = getRelevantAxes(m2t, axesHandles);
```

Comment:
@PeterPablo I could revert on that by I thought it might help in the future with handling multiple plots. For instance, you have the collection of relevant peer axes from which you could query the positions.